conan/tools/gnu/autotoolstoolchain.py

```diff
@@ -166,26 +167,25 @@ def generate(self, env=None, scope="build"):
         self.generate_args()
         VCVars(self._conanfile).generate(scope=scope)
 
+    def _shared_static_args(self):
+        args = []
+        try:
+            shared = self._conanfile.options.shared
+            if shared:
+               args.extend(["--enable-shared", "--disable-static"])
+            else:
+               args.extend(["--disable-shared", "--enable-static"])
+
+            if shared:
+                args.extend(["--with-pic" if self._conanfile.options.fPIC else "--without-pic"])
```

Comment:
But fpic isn't mandatory for shared libs? Also, if this toolchain might be used by Windows/mingw builds, is it possible that this fail?